### DISCUSSION TITLE
How to send a registration request?

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [logto-io/logto#6429] [Q&A]
I have a express.js  backend and a next.js frontend in my monorepo. And I want to keep my own login/registraion pages and I just want to send a request to my self-hosted logto behind the scenes. So that when the user completes the registration form, I can send an api call behind the scene to create the actual user in logto. And then make another api call to get the user and the token and so on.

Is it possible to use logto like that? I've been stuck for days trying to add logto to my existing architecutre without redirecting users outside to logto pages.

I currently have /auth/register endpoint which creates the user in my mongoDB. Is it possible to keep the same endpoint but make it create the user using logto? (without redirecting the user)

**Comment by wangsijie** [comment]
Hi, we are working on this, @xiaoyijun could you please share some details on the incoming feature?

**Reply by xiaoyijun** [reply]
Hi @TheMikeyRoss , it's not support, you'll always be redirected to the Logto sign-in page. Please refer to https://docs.logto.io/docs/concepts/sign-in-experience/#i-need-my-users-to-fill-in-their-credentials-in-my-app

If you just want to create users in your mongoDB, you can use a [webhook](https://docs.logto.io/docs/recipes/webhooks/) to accomplish that.

**Reply by TheMikeyRoss** [reply]
Oh ok so we're limited to the login/registration UI provided by logto?

**Reply by wangsijie** [reply]
We are also developing a feature called "Bring you own UI" which allows you to upload your own code to replace the Logto's sign in page. And this will be released soon.

**Reply by TheMikeyRoss** [reply]
@wangsijie Nice 👍 Are there any potential limitations? Because my login/registration page is highly customized using framer-motion and other packages that makes it flashy and smooth with my own custom animations
